src/plugins/unified_doc_viewer/public/components/doc_viewer_logs_overview/logs_overview.tsx

```diff
@@ -37,6 +38,7 @@ export function LogsOverview({
       <LogsOverviewHeader doc={parsedDoc} />
       <EuiHorizontalRule margin="xs" />
       <LogsOverviewHighlights formattedDoc={parsedDoc} flattenedDoc={hit.flattened} />
+      <LogsOverviewAIAssistant doc={hit} />
```

Comment:
We may need to rethink how we're registering and rendering this a bit once we do the work to support the generic logs flyout + the o11y flyout, but this can be done in steps and what we have here now is a good start.

Reply:
Correct, this is a temporary way of consuming this features since there is not differentiation yet for flavoured versions of Discover, once we have the resolution in place we can change this to be enabled only for a specific context.